src/arkit.cs

```diff
@@ -278,6 +287,55 @@ interface ARPlaneAnchor {
 			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
 			get;
 		}
+
+		[iOS (11,3)]
+		[Export ("geometry", ArgumentSemantic.Strong)]
+		ARPlaneGeometry Geometry { get; }
+	}
+
+	[iOS (11,3)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface ARPlaneGeometry : NSSecureCoding {
+		[Export ("vertexCount")]
+		nuint VertexCount { get; }
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Export ("vertices")]
+		IntPtr GetRawVertices ();
+
+		[Export ("textureCoordinateCount")]
+		nuint TextureCoordinateCount { get; }
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Export ("textureCoordinates")]
+		IntPtr GetRawTextureCoordinates ();
+
+		[Export ("triangleCount")]
+		nuint TriangleCount { get; }
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Export ("triangleIndices")]
+		IntPtr GetRawTriangleIndices ();
+
+		[Export ("boundaryVertexCount")]
+		nuint BoundaryVertexCount { get; }
+
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Export ("boundaryVertices")]
+		IntPtr GetRawBoundaryVertices ();
```

Comment:
Why are GetRawTextureCoordinates, GetRawTextureCoordinates and GetRawTextureCoordinates methods and not properties? 

At least in the diff we have:

`@property (nonatomic, readonly) const vector_float2 *textureCoordinates NS_REFINED_FOR_SWIFT;`

Reply:
The description of this PR (consistency with `ARFaceGeometry`), see https://github.com/xamarin/xamarin-macios/pull/3090 (properties were incorrect and obsoleted) and see Alex's comment https://github.com/xamarin/xamarin-macios/pull/3343#discussion_r164246635 (Sebastien made the same comment).